taskcluster/tf_tc-vars.sh

```diff
@@ -191,6 +191,10 @@ else
     BAZEL_EXTRA_FLAGS="--config=noaws --config=nogcp --config=nohdfs --config=nonccl --copt=-fvisibility=hidden"
 fi
 
+if [ "${OS}" = "Darwin" ]; then
+    BAZEL_EXTRA_FLAGS="${BAZEL_EXTRA_FLAGS} --macos_minimum_os 10.10"
```

Comment:
Isn't this going to set the value for all macOS-run builds? Said otherwise, do we want that to be set for iOS builds?